packages/components/src/Headline/Headline.tsx

```diff
@@ -52,9 +50,9 @@ const _Headline = ({
     <Heading
       level={Number(level)}
       {...props}
-      className={cn(classNames, 'text-[--color]', textAlign[align])}
+      className={cn(classNames, 'text-(--color)', textAlign[align])}
```

Comment:
instead of setting `--color` here can we not set the `color` prop as variable? e.g. `red-500` -> `--color-red-500`